ui/hooks/useCarouselManagement/useCarouselManagement.ts

```diff
@@ -39,60 +52,89 @@ export const useCarouselManagement = ({
-      // We need this to maintain the persisted dismissed state
-      const dismissedSweepstakesSlide = {
-        ...SWEEPSTAKES_SLIDE,
-        dismissed: true,
+      const fundSlide = {
+        ...FUND_SLIDE,
+        undismissable: hasZeroBalance,
       };
 
-      defaultSlides.push(dismissedSweepstakesSlide);
-    }
+      ///: BEGIN:ONLY_INCLUDE_IF(build-main,build-beta,build-flask)
+      defaultSlides.push(BRIDGE_SLIDE);
+      ///: END:ONLY_INCLUDE_IF
+      defaultSlides.push(CARD_SLIDE);
+      defaultSlides.push(CASH_SLIDE);
+      defaultSlides.push(MULTI_SRP_SLIDE);
+
+      defaultSlides.splice(hasZeroBalance ? 0 : 2, 0, fundSlide);
+
+      if (isRemoteModeEnabled) {
+        defaultSlides.unshift(REMOTE_MODE_SLIDE);
+      }
+
+      // Handle sweepstakes slide
+      const currentDate = testDate
+        ? new Date(testDate)
+        : new Date(new Date().toISOString());
+      const isSweepstakesActive = getSweepstakesCampaignActive(currentDate);
+
+      // Only show the sweepstakes slide if:
+      // 1. Not in test mode
+      // 2. Sweepstakes campaign is active
+      // 3. Slide has not been dismissed by user
+      if (!inTest && isSweepstakesActive && !isSweepstakesSlideDismissed) {
+        const newSweepstakesSlide = {
+          ...SWEEPSTAKES_SLIDE,
+          dismissed: false,
+        };
+        defaultSlides.unshift(newSweepstakesSlide);
+      } else if (existingSweepstakesSlide?.dismissed) {
+        // Add the sweepstakes slide with the dismissed state preserved
+        // We need this to maintain the persisted dismissed state
+        const dismissedSweepstakesSlide = {
+          ...SWEEPSTAKES_SLIDE,
+          dismissed: true,
+        };
+
+        defaultSlides.push(dismissedSweepstakesSlide);
+      }
+
+      // dispatch(updateSlides(defaultSlides));
+
+      // Handle Contentful Data
+      try {
+        const contentfulSlides = await fetchCarouselSlidesFromContentful();
+        console.log('contentfulSlides', contentfulSlides);
+        const checkedContentfulSlides = contentfulSlides
+          .map((slide) => {
+            const existing = slides.find(
+              (s: CarouselSlide) => s.id === slide.id,
+            );
+            return {
+              ...slide,
+              dismissed: existing?.dismissed ?? false,
+            };
+          })
+          .filter((slide) =>
+            isActive(slide, testDate ? new Date(testDate) : new Date()),
+          );
+        console.log('checkedContentfulSlides', checkedContentfulSlides);
+        const mergedSlides = [...defaultSlides, ...checkedContentfulSlides];
+        dispatch(updateSlides(mergedSlides));
+      } catch (err) {
+        console.error('Failed to fetch Contentful slides:', err);
+        dispatch(updateSlides(defaultSlides));
+      }
+    };
 
-    dispatch(updateSlides(defaultSlides));
-  }, [dispatch, hasZeroBalance, isRemoteModeEnabled, slides, testDate, inTest]);
+    loadSlides();
```

Comment:
Nice async effect, lets make sure we cleanup/remove the console.logs and comments.